CONTRIBUTING.md

```diff
@@ -78,6 +78,8 @@ We need people to identify and research incidents and ensure that the current re
 
 To contribute to the incident reports, you will need to identify events where police used excessive force or abused their power during the George Floyd protests.
 
+**Note:** We do not doxx anyone, including police officers, unless it's linking to a news article with officially recognized information. If you'd like to give us doxxing information, message the mods of the reddit and we'll file that information which we can provide privately to relevant parties. We want to avoid being the cause of a witch hunt.
```

Comment:
I think at least for right now we want to just avoid this altogether. I don't disagree with the idea of trying to identify criminal police, but we also don't need a boston bomber situation developing where we get some poor guy attacked by a lynch mob because of a mis-identification.

If it links to a news article identifying someone, that's fine to add as additional context to an existing report.

Reply:
Ah, was just adding in what @ubershmekel suggested [here](https://github.com/2020PB/police-brutality/pull/247#issuecomment-639227547). I'll do another PR that doesnt include the "send to the mods" bit